### PR TITLE
Added an amazing idea to the already exisiting ones

### PR DESCRIPTION
**Pull request by ADI10HERO**
Added one more amazing idea, will propose more in future !!

### CODE REVIEW

**Comment by ADI10HERO** [comment]
Merge or close being on the wait list is frustrating xD

**Comment by melanierichards** [comment]
Hi @ADI10HERO, thanks for sending a ping! I think I missed the message for your PR the first time around somehow, maybe accidentally marked all as read. 🤔

Thanks for submitting ideas! Leaving one small comment

File: README.md [hunk 1]
```diff
@@ -83,4 +83,5 @@ But what if you don't know what kind of website to "just build"? The following i
 * Create a custom 404 page
 * Create a random number generator for picking lottery numbers ([Guidelines](docs/lottery-numbers.md))
 * Go to the [new repo page](https://github.com/new) on Github. Make something inspired by the randomly-generated repo name they give you
-* Recreate the title sequence of a favorite show or movie using CSS. (Example: [Stranger Things](https://github.com/wbobeirne/stranger-things))
\ No newline at end of file
+* Recreate the title sequence of a favorite show or movie using CSS. (Example: [Stranger Things](https://github.com/wbobeirne/stranger-things))
+* Create a frontend with historic or futuristic theme (or any theme accordingly).
```
Comment (melanierichards) [inline README.md]: Instead of frontend, maybe say user interface? Or you could rephrase as "Create a site theme with historic or futuristic influence"?

Comment (ADI10HERO) [reply README.md]: Yes Sure :)

**Comment by ADI10HERO** [comment]
I have another idea to add too :P

**Comment by melanierichards** [comment]
Merged this one, go for it! 😀

**Comment by ADI10HERO** [comment]
Thanks 😌
Sure!

**Pull request merged**